railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -52,7 +52,7 @@ Rails.application.configure do
   <%- end -%>
   # Assume all access to the app is happening through a SSL-terminating reverse proxy.
   # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
-  # config.assume_ssl = true
+  config.assume_ssl = true
```

Comment:
Assuming ssl is less secure by default.
It makes it easier to accidentally miss that you forgot ssl.

Also, I do not understand why this is necessary.
It seems Rack is already assuming ssl in a wide range of situations https://github.com/rack/rack/blob/4795831a0a310c2d31102749e551b38faab6401f/lib/rack/request.rb#L254

And the argument in the commit message says Cloudflare, which is covered by Rack (Cloudflare sets a header https://developers.cloudflare.com/fundamentals/reference/http-headers/#x-forwarded-proto and Rack checks it https://github.com/rack/rack/blob/4795831a0a310c2d31102749e551b38faab6401f/lib/rack/request.rb#L724)